plotly/tools.py

```diff
@@ -251,7 +251,7 @@ def reset_config_file():
 
 ### embed tools ###
 
-def get_embed(file_owner_or_url, file_id=None, width="100%", height=525):
+def get_embed(file_owner_or_url, file_id=None, share_key=None, width="100%", height=525):
```

Comment:
Does this mean users need to *know* the share key ahead of time? I'm not sure we provide a way to get at that programmatically, so that forces them to need to click around in plot.ly.

@chriddyp thoughts on this one? It's gonna require a quick api hit to get the share key, but I think it'll be easier to work with.